Add February Collab Cafe notes (#711)
jupyterhub/team-compass: docs/meetings/collab-cafe/2024.md

# 2024 Collaboration Cafe Notes Archive

## 2024-02-20

### Check-in :raising_hand:

- Min / @minrk / Simula
- Sarah / @sgibson91 / 2i2c
- Samuel / @sgaist / Idiap Research Institute
- Kirstie / @KirstieJane / The Alan Turing Institute
- Simon / @manics / University of Dundee
- Steve / @stevejpurves / Curvenote | MystMarkdown

### Introduce yourself! :wave:

_If you are new to the meeting, add your name below and you can introduce yourself at the start of the meeting._

- Kirstie Whitaker - Programme Director for Tools, Practices and Systems at The Alan Turing Institute. Founder of The Turing Way. Very interested in open source governance (EDI, accountability, transparency, project management etc)

### Agenda :clock2:

- **Kirstie** (15mins): CZI Essential Open Source Software for Science (EOSS) Diversity, Equity and Inclusion (DEI) funding revamp proposal

>   - <https://docs.google.com/document/d/1pSU9kG_XDOSTF4JcbIt18hep1rCHIjuUYuYi5mg0q-0/edit>

Reviewer: This doc is publicly editable without a login. Is it still OK to share this and the other gdoc link later in the notes on the team compass website?

Author: @KirstieJane?

Author: The 2nd doc is suggestion only, not editing. So I think that's fine.

>   - <https://docs.google.com/document/d/1pSU9kG_XDOSTF4JcbIt18hep1rCHIjuUYuYi5mg0q-0/edit>

Reviewer 2: ```suggestion
  - <https://docs.google.com/document/d/1pSU9kG_XDOSTF4JcbIt18hep1rCHIjuUYuYi5mg0q-0/edit?usp=sharing>
```

I changed the setting on the doc so its now suggestion only (I hope?)

Thank you for catching @manics !

Author: Yep, it's suggestion only when I visit it!

        - Variety of interaction in the council, and there isn't a clear set of expectations in what the council is doing
        - different types of leadership, technical decisions, community management, organisational leadership, paid development time
- **Steve** (15mins): Contributing a new content/repo provider (MECA) to BinderHub/mybinder.org - how to?
  - Notebooks Now! project run by AGU, funded project to pilot using Notebooks to build interactive, computational article, and formally publish them
  - How to integrate with XML format (specifically JATS used by journals, also comes with a zip file MECA) so that Notebook-based publications can be accepted
  - MECA is an existing NISO standard (<https://www.niso.org/standards-committees/meca>) , modify JATS to also understand Notebooks, and bundle MECA with the binder requirements for launching. Completely disassociate from a GitHub repo.
  - <https://agu.curve.space> - demo website showing what the journal might look like
  - New repo-provider "MECA for binder", how to get code into repo2docker? how to deploy it onto mybinder.org?
    - PRs:

>       - #1335 on repo2docker repo
>       - #1824 on binderhub repo

Reviewer: ```suggestion
      - [#1335](https://github.com/jupyterhub/repo2docker/pull/1335) on repo2docker repo
      - [#1824](https://github.com/jupyterhub/binderhub/pull/1824) on binderhub repo
```

  - We don't have much of a review process at the moment and it is hard to allocate time and expertise to the review.
    - Do we make the decision in the meeting? How many people need to say "yes" for it to be a real "yes"!
    - Process isn't written down.
  - The project is very technical and there are a lot of information siloes.
    - Some folks don't feel confident enough to review (and that might be truely that they don't have the expertise! Even if they're leaders in the community.)
  - Identifying reviewers in advance can be hard.
  - Some other projects allocate a reviewer (often by a bot) and then they know that it is their responsibility to take the review forwards.
    - They become the point person so even if they don't know the right details _they_ can go and get the person who has the expertise.
  - Additional challenge around repo2docker as a specification - might have additional maintenance burden if edits are incorporated there.
  - Bias towards the more recent tasks and the easier or noiser tasks for reviewers.
    - Potentially needs a specific meeting (or async equivalent) to set the priorities
  - Hard to figure out how to reach consensus

>   - "Yuvi and Sarah spoke to a governance expert on some of these things, I think section C here is particularly relevant": <https://docs.google.com/document/d/1WJMRR7ggIivmccrAjIgFWwLt9NYpzrn87ku-4GAFbnw/edit?pli=1#heading=h.ob57fk58k5ss>

Author: ```suggestion
  - "Yuvi and Sarah spoke to a governance expert on some of these things, I think section C here is particularly relevant": <https://docs.google.com/document/d/1WJMRR7ggIivmccrAjIgFWwLt9NYpzrn87ku-4GAFbnw/edit?usp=sharing>
```
